asciidoc/courses/llm-fundamentals/modules/3-intro-to-langchain/lessons/1-langchain/questions/2-languages.adoc

```diff
@@ -0,0 +1,23 @@
+[.question]
+= Programming Languages
+
+What programming languages can you build a Langchain application in?
+
+* [ ] C#
+* [ ] Java
+* [*] JavaScript
+* [*] Python
+* [*] TypeScript
+
+[TIP,role=hint]
+.Hint
+====
+Langchain offers libraries written in two programming languages.
```

Comment:
I get that only two programming languages are supported but there are 3 answers. If you werent familiar with Typescript / JavaScript you wouldnt know this.